worker-languagedetection-container/testcases/automated/CAF_4001/hmong.yaml.testcase

```diff
@@ -34,39 +34,39 @@ expectedOutputData:
       DetectedLanguage1_Code:
         action: "replace"
         values:
-        - data: "hmn"
+        - data: "unknown"
       DetectedLanguage1_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "99"
+        - data: "32"
```

Comment:
Significant change here, cannot identify hmong anymore. If it is not supported then remove this testcase and make sure that hmong is not referred to as a supported language.